### PR TITLE
Add chevron to Safari and Firefox dropdowns

### LINKED ISSUE #83756: Draw custom drop-down picker (Desktop, Web)
Web on the left, electron on the right:
image

### PR DESCRIPTION
**Pull request by miguelsolorio**
Related #83556
Fixes #83756

This adds the chevron to all of our dropdowns in Safari/Firefox:

image

### CODE REVIEW

**Review by bpasero — CHANGES_REQUESTED** [review]
Moving to November, as I think this is not too critical for now.

image

Looks very cool! Shouldn't we do this simply for Chrome/Electron as well then?

My only other comment would be that I would rather not have workbench-contribution specific CSS rules (like debug) in the core. Rather move them to debug or their respective area.

**Comment by misolori** [comment]
@bpasero moved the styles out to their respective areas (Debug and Panel)

**Comment by bpasero** [comment]
@misolori any objections using this on Chrome as well?

**Comment by misolori** [comment]
@bpasero sorry I missed that part about your comment, I don't see any objections and it makes sense to make it consistent across all of the products. I'll update this PR to reflect this.

**Comment by bpasero** [comment]
Cool.

**Comment by bpasero** [comment]
@misolori I see a couple of issues:
* you cannot click the arrow to open the dropdown anymore
* the arrow overflows with the content
image

* there is some cheese in the panel (if used in Chrome)

image

**Comment by bpasero** [comment]
@misolori I pushed https://github.com/microsoft/vscode/pull/83793/commits/863defb0a84d76101d7f71018a4c735ef597932d to make it a general thing, feel free to revert if you think that does not work

**Comment by misolori** [comment]
I've updated the styles so it cleans it up in all browsers/platforms. I also had to set the appearance to `none` in order to remove the browser dropdown arrows.

## FireFox
image

## Chrome
image

## Safari
image

## OSS
image

**Comment by misolori** [comment]
> you cannot click the arrow to open the dropdown anymore

I need to look into this more, it seems like in the debug dropdown the `selection` input doesn't extend out to the icon.

**Comment by bpasero** [comment]
Ok I can review once you figured that out if possible.

**Comment by misolori** [comment]
@bpasero dropdown chevrons should now be clickable

File: src/vs/workbench/browser/media/style.css [hunk 1]
```diff
@@ -174,6 +174,33 @@ body.web {
 	cursor: default;
 }
 
+/* Custom Dropdown (select) Arrows */
+
+.monaco-workbench select {
+
```
Comment (bpasero) [inline src/vs/workbench/browser/media/style.css]:
```suggestion

```

Comment (bpasero) [reply src/vs/workbench/browser/media/style.css]: Dangling empty line.

File: src/vs/workbench/browser/parts/panel/media/panelpart.css [hunk 2]
```diff
@@ -110,7 +110,12 @@
 	text-align: center;
 	display: inline-block;
 	box-sizing: border-box;
-	}
+}
+
+/* Dropdown */
+.monaco-workbench .panel .monaco-action-bar .select-container .monaco-select-box {
```
Comment (bpasero) [inline src/vs/workbench/browser/parts/panel/media/panelpart.css]: Shouldn't this move into the rule we already have for `.monaco-select-box` below?

**Review by bpasero — APPROVED** [review]
LGTM with some feedback. Also, I did only test macOS, can you check Windows and Linux too?

File: src/vs/workbench/browser/parts/panel/media/panelpart.css (hunk 2 above)
Comment (misolori) [reply src/vs/workbench/browser/parts/panel/media/panelpart.css]: This was specific to the panel, but I just double-checked and this rule doesn't need to be here anymore so I'll remove it.

**Comment by misolori** [comment]
✅ Windows (OSS, Chrome, Firefox)
✅ Linux (OSS, Chrome, Firefox)

**Pull request merged**